providers/google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/compute/strategy/CreateNodesWithGroupEncodedIntoNameThenAddToSet.java

```diff
@@ -186,6 +202,16 @@ private void getOrCreateFirewalls(GoogleComputeEngineTemplateOptions templateOpt
          operationDone.apply(operation);
          checkState(operation.get().httpErrorStatusCode() == null, "Could not insert firewall, operation failed %s",
                operation);
+
+         // Add tags for firewalls
+         tags.add(name);
+      }
+      templateOptions.tags(tags);
+   }
+
+   public static void validateFirewall(Firewall firewall, URI networkUri) {
```

Comment:
Does it make more sense for the signature to be validateFirewall(Firewall firewall, Network network)?